src/styles/normalize.scss

```diff
@@ -0,0 +1,6 @@
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
```

Comment:
just add the class "page" to the body and move your styles here. Normalize file is generally used for fonts etc.

Reply:
Thanks. Fixed